gatsby-node.js

```diff
@@ -42,31 +42,50 @@ const SSR_getSummit = async (baseUrl) => {
   };
 
   return await axios.get(
-      `${baseUrl}/api/public/v1/summits/current`,
-      { params }
+    `${baseUrl}/api/public/v1/summits/current`,
+    { params }
   )
-      .then(({data}) => data)
-      .catch(e => console.log('ERROR: ', e));
+    .then(({ data }) => data)
+    .catch(e => console.log('ERROR: ', e));
 };
 
 const SSR_getEvents = async (baseUrl, summitId, accessToken, page = 1, results = []) => {
   return await axios.get(
-      `${baseUrl}/api/v1/summits/${summitId}/events/published`,
-      {
-        params: {
-          access_token: accessToken,
-          per_page: 50,
-          page: page,
-          expand: 'slides, links, videos, media_uploads, type, track, track.allowed_access_levels, location, location.venue, location.floor, speakers, moderator, sponsors, current_attendance, groups, rsvp_template, tags',
-        }
-      }).then(({data}) => {
-    if (data.current_page < data.last_page) {
-      return SSR_getEvents(baseUrl, summitId, accessToken, data.current_page + 1, [...results, ...data.data]);
-    }
+    `${baseUrl}/api/v1/summits/${summitId}/events/published`,
+    {
+      params: {
+        access_token: accessToken,
+        per_page: 50,
+        page: page,
+        expand: 'slides, links, videos, media_uploads, type, track, track.allowed_access_levels, location, location.venue, location.floor, speakers, moderator, sponsors, current_attendance, groups, rsvp_template, tags',
+      }
+    }).then(({ data }) => {
+      if (data.current_page < data.last_page) {
+        return SSR_getEvents(baseUrl, summitId, accessToken, data.current_page + 1, [...results, ...data.data]);
+      }
 
-    return [...results, ...data.data];
-  })
-      .catch(e => console.log('ERROR: ', e));
+      return [...results, ...data.data];
+    })
+    .catch(e => console.log('ERROR: ', e));
+};
+
+const SSR_getProfileSettings = async (baseUrl, page = 1, results = {}) => {
```

Comment:
please rename to something more meaningful
i know that we are using this for the profile .. but
in the bottom line we are getting the components of the current openstack release. so it could be used for something else on the future
lets rename it to something like SSR_getCurrentReleaseComponents